File: cfgs/pythoncfgs/MAXIMIZE_SUM_ARRII.py
Nodes
1;statement;fun_1();8;
2;statement;foo_1 = 0;9;
3;conditional;fun_1(foo_1);10;
4;statement;return foo_1;12;
5;statement;foo_1 += foo_2[foo_3] * foo_4;11;
6;exit;;
Edges
0;2;
1;3;
2;4;5;
3;6;
4;3;
5;

File: cfgs/java_cfgs_foo/MAXIMIZE_SUM_ARRII.java
Nodes
1;special;;;
2;statement;fun_1(foo_1);14;
3;statement;int sum = 0;15;
4;statement;int i = 0;16;
5;conditional;foo_1 < foo_2;17;
6;statement;sum += foo_1[foo_2] * foo_3;18;
7;statement;i++;18;
8;statement;return foo_1;19;
9;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;8;
5;7;
6;5;
7;9;
8;
